Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs

```diff
@@ -41,12 +41,10 @@ protected override void OnCreate(Bundle bundle)
 			base.OnCreate(bundle);
 
 #if TEST_EXPERIMENTAL_RENDERERS
-			// CollectionView lets us test CollectionView stuff until it's officially released
-			Forms.SetFlags("FastRenderers_Experimental"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
-#else
 			// Fake_Flag is here so we can test for flag initialization issues
-			// CollectionView lets us test CollectionView stuff until it's officially released
-			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental" */); 
+			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
+#else
+			Forms.SetFlags("UseLegacyRenderers"/*, "CollectionView_Experimental", "Shell_Experimental" */);
```

Comment:
`UseLegacyRenderers` -> `NonFastRenderers_Legacy`? Cuz not all renderers have a legacy, right? Just those with fast versions right? And, I hate to ponder, but if we introduce new fast renderers for those yet to be converted then we might consider enumerating the renderers explicitly..

Reply:
The idea is that we'll always set a flag of some sort. Since I removed the flag from TEST_EXPERIMENTAL_RENDERERS and added a flag when we're not on fast renderers, I just moved the fake flag.

Reply:
If it's needed, we can add more fine-grained control. There are still workarounds for individual controls. For example, a developer could use an `ExportRenderer` and explicitly replace all Buttons with the old `ButtonRenderer` instead of using this flag. And yes, we may or may not be putting any new fast renderers under a new experimental flag. That's TBD. Not all renderers have a legacy at this point, but I like to think that this flag will happily encapulate them all when they do.

Reply:
Roger.